### ISSUE TITLE
Unexpected error when trying to get symbols

### ISSUE THREAD

**Issue by txeriff** [Microsoft/AL#3590]
Hi, 
Sending request to https://fc80ee95b1d7:8080/nav/dev/packages?publisher=Microsoft&appName=System&versionText=11.0.0.0
I the the follwoing error:
the link protocol could not be carried out due to an unexpected packet format.

**Comment by kalberes** [comment]
IS this a Nav 2018 issue?

**Comment by txeriff** [comment]
Yes it is. Docker image

**Comment by kalberes** [comment]
Which version of the AL extension are you using? Also anything in the eventviewer for the server on the docker.

**Comment by txeriff** [comment]
Hi, good morning, 

I´m running .vsix from docker image, it says 0.12.28462

It does not show anything in the powershell window, just VSCode screen:
This is the error I get ( I just removed spanish lang from vstudio code, but still some spanish error msgs)
(replaced server and usernames with XXX)

_[2018-09-10 09:31:38.12] Using reference symbols cache path: c:\Users\XXX\Documents\AL\ALProject2\.alpackages
[2018-09-10 09:31:38.19] Sending request to https://fc80ee95bXXX:8080/nav/dev/packages?publisher=Microsoft&appName=Application&versionText=11.0.0.0
[2018-09-10 09:31:38.33] Sending request to https://fc80ee95bXXX:8080/nav/dev/packages?publisher=Microsoft&appName=System&versionText=11.0.0.0
[2018-09-10 09:31:40.65] Error: Error al enviar la solicitud.
Se ha terminado la conexión: Error inesperado de envío.
No se pudo llevar a cabo el protocolo de enlace debido a un formato de paquete inesperado.

[2018-09-10 09:31:40.66] Error: Error al enviar la solicitud.
Se ha terminado la conexión: Error inesperado de envío.
No se pudo llevar a cabo el protocolo de enlace debido a un formato de paquete inesperado._

**Comment by txeriff** [comment]
No, cos i havent got the symbols.

**Comment by kalberes** [comment]
And have you  downloaded them, would you then be able to publish it?

**Comment by txeriff** [comment]
No, I click download symbols, then I get such error. I cannot go further. I´ve tried also with vsix from marketplace but same error. 
It must be some error with route or something.

**Comment by kalberes** [comment]
Hmm I am clueless. Looks like your docker image has some issues? Is it an option to merge to the newest one released?

**Comment by txeriff** [comment]
![prueba](https://user-images.githubusercontent.com/43034938/45310007-6bb15000-b525-11e8-872d-259b338d9397.png)

**Comment by txeriff** [comment]
Hi again. I changed port from 8080 to 443. It does not say anywhere, I had to figure out.
[2018-09-12 17:51:23.75] Sending request to https://fc80ee95b1d7/nav/dev/apps?SchemaUpdateMode=synchronize
[2018-09-12 17:51:26.16] The request for path /nav/dev/apps?SchemaUpdateMode=synchronize failed with code NotFound. Reason: Not Found

now I get these errors. any ideas?

**Comment by kalberes** [comment]
What is the DeveloperServicesPort in your customsettings,config in the server?
By default it is 7049. Besides this port, the DeveloperServicesEnabled should be set to true.
If it is a different port number then you should specify the same port in the launch.config. "port:   "
Also for your server settings in the launch.json it is enough to specify
"server": "http://fc80ee95b1d7"
